packages/tools/devtools/devtools/src/index.ts

```diff
@@ -187,6 +189,14 @@ function mapContainerProps(
 	};
 }
 
+/**
+ * Gets the Devtools singleton if it has been initialized, otherwise returns `undefined`.
```

Comment:
Nit (for API discoverability):
```suggestion
 * Gets the Devtools singleton if it has been {@link initializeFluidDevtools | initialized}, otherwise returns `undefined`.
```

Reply:
(Doesn't look like this suggestion was applied. Not a strict requirement, but I do think it would be nice for discoverability).

Reply:
updated